src/literal.ts

```diff
@@ -1,13 +1,36 @@
 'use strict'
-import ClassOrder from './class-order'
 import NamedNode from './named-node'
 import Node from './node-internal'
 import XSD from './xsd'
+import { ValueType } from './types'
+import classOrder from './class-order'
 
+/**
+* An RDF literal node, containing something different than a URI.
+* @link https://rdf.js.org/data-model-spec/#literal-interface
+*/
 export default class Literal extends Node {
-  constructor (value, language, datatype) {
+  /**
+   * The language for the literal
+   */
+  lang!: string
+
+  /**
+   * The literal's datatype as a named node
+   */
+  datatype!: NamedNode
+
+  termType: string
```

Comment:
Given that this can only be one string, this might be better?
```suggestion
  termType: 'Literal'
```